src/modules/documentExecution/dashboard/dataset/DatasetEditorDataTab/DatasetEditorPreview.vue

```diff
@@ -32,17 +32,19 @@ import Message from 'primevue/message'
 import mainDescriptor from '@/modules/workspace/WorkspaceDescriptor.json'
 import workspaceDataPreviewDialogDescriptor from '@/modules/workspace/views/dataView/dialogs/WorkspaceDataPreviewDialogDescriptor.json'
 import mainStore from '../../../../../App.store'
+import dashboardStore from '@/modules/documentExecution/dashboard/Dashboard.store'
 
 import deepcopy from 'deepcopy'
 
 export default defineComponent({
     name: 'kpi-scheduler-save-dialog',
     components: { Dialog, DatasetPreviewTable, Message },
-    props: { visible: { type: Boolean }, propDataset: { type: Object }, previewType: String },
+    props: { visible: { type: Boolean }, propDataset: { type: Object }, previewType: String, dashboardId: { type: String, required: true } },
     emits: ['close'],
     setup() {
         const store = mainStore()
-        return { store }
+        const dashStore = dashboardStore()
+        return { store, dashStore }
```

Comment:
please do remember to avoid using composition api setup for the store. Use pinia helpers instead